apps/trading/pages/client-router.tsx

```diff
@@ -26,10 +26,23 @@ const LazyPortfolio = dynamic(() => import('../client-pages/portfolio'), {
 
 export enum Routes {
   HOME = '/',
-  MARKETS = '/markets',
+  SKELETON_MARKET = '/markets',
+  MARKET = '/markets/:marketId',
+  MARKETS = '/markets/all',
   PORTFOLIO = '/portfolio',
+  LIQUIDITY = 'liquidity/:marketId',
 }
 
+export const Links = {
```

Comment:
suggestion (non blocking): could type this with something like `{ [R in Routes]: () => string }` so that we don't miss a link function for a route